webapp/graphite/render/functions.py

```diff
@@ -2631,12 +2631,14 @@ def threshold(requestContext, value, label=None, color=None):
 
   return [series]
 
-def transformNull(requestContext, seriesList, default=0):
+def transformNull(requestContext, seriesList, default=0, keySeries=None):
   """
-  Takes a metric or wild card seriesList and an optional value
-  to transform Nulls to. Default is 0. This method compliments
-  drawNullAsZero flag in graphical mode but also works in text only
-  mode.
+  Takes a metric or wild card seriesList and an optional value to transform
```

Comment:
Some grammatical typos here. Some are inherited from the previous version, some are new. Would you mind fixing these?

`wild card` should be `wildcard`